libs/blocks/global-navigation/global-navigation.js

```diff
@@ -317,7 +317,9 @@ class Gnav {
       await yieldToMain();
       await task();
     }
-
+    // Dispatching an event when gnav loads and is visible
+    window.dispatchEvent(new CustomEvent('feds:nav.ready'));
```

Comment:
```suggestion
    window.dispatchEvent(new CustomEvent('feds:nav:ready'));
```

Reply:
On another note, since it's a milo block, should we dispatch `milo:globalnav:ready` ? That follows the other eventlisteners such as `milo:modal:closed`